bigtop-packages/src/common/hbase/do-component-build

```diff
@@ -36,13 +36,19 @@ if [ $HOSTTYPE = "aarch64" ] ; then
    sed -i '/<version>1.5.0-alpha.6<\/version>/,+1d' ./pom.xml
    sed -i '/<artifactId>asciidoctorj-pdf<\/artifactId>/a\<version>1.5.0-alpha.6<\/version>\n</dependency>\n<dependency>\n<groupId>org.jruby<\/groupId>\n<artifactId>jruby-complete<\/artifactId>\n<version>9.1.8.0<\/version>\n<\/dependency>' ./pom.xml
 fi
-mvn -DskipTests -Dslf4j.version=1.6.1         \
-    -Dhadoop-two.version=$HADOOP_VERSION      \
-    -Dzookeeper.version=$ZOOKEEPER_VERSION    \
-    -Dcheckstyle.skip=true                    \
-    ${MAVEN_ADDITIONAL} \
-    install site assembly:single "$@"
+
+MAVEN_PARAMS=" -DskipTests "
+MAVEN_PARAMS+=" -Dslf4j.version=1.7.25 "
+MAVEN_PARAMS+=" -Dhadoop-two.version=${HADOOP_VERSION} "
+MAVEN_PARAMS+=" -Dzookeeper.version=${ZOOKEEPER_VERSION} "
+MAVEN_PARAMS+=" -Dcheckstyle.skip=true "
+
+# separate site and assembly:single to avoid assembly issues.
```

Comment:
May I know what's the issue here that require the separation?

Reply:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.0.0:single (default-cli) on project hbase-assembly: Failed to create assembly: Error adding file 'org.apache.hbase:hbase-common:test-jar:tests:2.2.2' to archive: /home/centos/bigtop/build/hbase/rpm/BUILD/hbase-2.2.2/hbase-common/target/test-classes isn't a file. -> [Help 1]
```
The error was discussed in [ML](https://lists.apache.org/thread.html/0c39364a2949fab134107eaf164723e09d895b3eb2c16610b841e033%40%3Cuser.hbase.apache.org%3E) and [HBASE-21513](https://issues.apache.org/jira/browse/HBASE-21513). While the root cause seems to be unclear, `site` and `assembly:sinble` are separated as a workaround in the create-release script of HBase side. It worked for me too.

Reply:
Got it. Thanks for the explanation. I suggest to add HBASE-21513 in the comment so other users can find the root cause themselves. We used to have the format like this:
```
# HBASE-21513: separate site and assembly:single to avoid assembly issues.
```

Reply:
Thanks. I updated the comment as you suggested.